src/main/java/com/esri/samples/portal/integrated_windows_authentication/README.md

```diff
@@ -0,0 +1,48 @@
+# Integrated windows authentication
+
+Use Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+
+![](IntegratedWindowsAuthentication.png)
+
+## Use case
+
+Integrated Windows Authentication, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+
+## How to use the sample
+
+1. Enter the URL to your IWA-secured portal into the text field.
+2. Select either the `Search Public Portal` button (which will search for portals on www.arcgis.com) or `Search IWA Secured Portal` (which will search your IWA-secured portal), to search for web maps stored on the portal.
+3. If you chose to search a secured portal, you will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password.
+4. If you authenticate successfully, portal item results will display in the list.
+5. Select a web map item to display it in the map view.
+
+## How it works
+
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username@domain and password) if a secure resource is encountered.
```

Comment:
I think this should be more clear and should be rewritten to take the user through a step by step API approach with reference to specific Java methods where appropriate, as we do in other samples. This description is a little broad for this section.

Reply:
Revamped this and added more detail. Let me know what you think.